src/chigame/games/templatetags/tags.py

```diff
@@ -0,0 +1,11 @@
+from django import template
+
+from chigame.games.models import Game
+
+register = template.Library()
```

Comment:
Please add comments explaining what is happening in this code here. 
As template tags are something not covered in the course IIRC, thus explaining more of this code would be good for the rest of the chigame devs.